src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,15 @@
+
+server.port=8443
+
+
+# postgres db connection
+# todo: use environment system variables
+spring.datasource.url=jdbc:postgresql://localhost:5432/arena-db
+spring.datasource.password=postgres
+spring.datasource.username=postgres
+spring.datasource.driver-class-name=org.postgresql.Driver
+
+
+# hibernate and jpa
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
```

Comment:
Add this to gitignore, preferably only leave the config keys and omit the values. Not too critical but best practice